modules/core/include/opencv2/core/mat.hpp

```diff
@@ -192,6 +194,7 @@ class CV_EXPORTS _InputArray
     UMat getUMat(int idx=-1) const;
     void getMatVector(std::vector<Mat>& mv) const;
     void getUMatVector(std::vector<UMat>& umv) const;
+    void getGpuMatVector(std::vector<cuda::GpuMat>** gpumv) const;
```

Comment:
I think it will be better to use `std::vector<cuda::GpuMat>& gpumv` to be consistent with other methods.